BenadrylAutomation/src/test/java/com/jnj/testcases/HeaderPageTest.java

```diff
@@ -19,15 +20,62 @@ public void setup() throws InterruptedException {
 		launchApplication();
 	}
 
-	@Test(priority = 1, description = "Verify main menu links", dataProvider = "headerMainMenu", dataProviderClass = HeaderProvider.class, enabled = true)
+	@Test(priority = 1, dataProvider = "headerMainMenu", dataProviderClass = HeaderProvider.class, enabled = true, groups = "HeaderPage")
```

Comment:
@goutamnaik007 remove enabled=true. not needed bydefault it will run all the methods

Reply:
@vaibhavNagvekarSJI removed enabled flag as suggested.